CMakeLists.txt

```diff
@@ -3,12 +3,24 @@
 # CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
 
 cmake_minimum_required(VERSION 2.8.7)
+PROJECT(GazeboYARPPlugins)
 
-PROJECT(GazeboYarpPlugins)
+# Project version
+set(VARS_PREFIX ${PROJECT_NAME})
```

Comment:
If `VARS_PREFIX` is used just in the next few lines, I think it would cleaner and more maintainable if we just used ${PROJECT_NAME} whenever we use `${VARS_PREFIX}`.